hlr_test.go

```diff
@@ -16,6 +16,40 @@ var hlrObject []byte = []byte(`{
   "statusDatetime":"2015-01-04T13:14:09+00:00"
 }`)
 
+var hlrListObject = []byte(`{
+	"offset": 0,
+    "limit": 20,
+    "count": 2,
+    "totalCount": 2,
+    "links": {
+        "first": "https://rest.messagebird.com/hlr/?offset=0",
+        "previous": null,
+        "next": null,
+        "last": "https://rest.messagebird.com/hlr/?offset=0"
+    },
+    "items": [
+	{
+		"id":"27978c50354a93ca0ca8de6h54340177",
+	    "href":"https:\/\/rest.messagebird.com\/hlr\/27978c50354a93ca0ca8de6h54340177",
+	    "msisdn":31612345678,
+	    "network":20406,
+	    "reference":"MyReference",
+	    "status":"sent",
+	    "createdDatetime":"2015-01-04T13:14:08+00:00",
+	    "statusDatetime":"2015-01-04T13:14:09+00:00"
+	},
+	{
+		"id":"27978c50354a93ca0ca8de6h54340177",
+	    "href":"https:\/\/rest.messagebird.com\/hlr\/27978c50354a93ca0ca8de6h54340177",
```

Comment:
Not sure if escaping slashes here makes sense.
https://play.golang.org/p/6aAUYzV1ac